guides/mat-elevation.md

```diff
@@ -0,0 +1,33 @@
+# mat-elevation
+mat-elevation exists to give you separation between elements.
+All material design elements have resting elevations.
+[`Material design`](https://material.io/guidelines/material-design/elevation-shadows.html)
+explains how best to use elevation.
+
+
+Elevation is implemented with a class, simply adding the class `mat-elevation-z#` where # is the elevation number you want, 0-24.
+
+## Example
+<!-- example(elevation-overview) -->
+
+
+## Mixins
+In order to use the mixin for Elevation you must 
+`@import '~@angular/material/core/style/_elevation.scss';`
```

Comment:
I think this should be `~@angular/material/theming` now that they're using a flat release structure (#3776)